### ISSUE TITLE
Fix Django performance issues on MacOS

### ISSUE THREAD

**Issue by rachelekm** [azavea/echo-locator#436]
We're experiencing some performance issues loading pages within the new Django `/admin` site. Since MacOS users are the only ones experiencing the inconsistent performance, and we frequently bump into conflicts with Docker on MacOS, we're assuming this is exclusively an issue with Docker on MacOS. Some clarifying points: the `taui` frontend site is still performing as expected; when a page loads slowly we notice duplicate calls to static pages/assets in the console as well as `[CRITICAL] WORKER TIMEOUT` messages from the database, although not always; these issues were still present after updates to Docker Engine v20.10.13 and Docker Compose v1.29.2.

**Comment by maurizi** [comment]
this may be a helpful read https://www.docker.com/blog/speed-boost-achievement-unlocked-on-docker-desktop-4-6-for-mac/

**Comment by rachelekm** [comment]
Closing this issue with semi-inconclusive results. For extra background, requests to `/admin/ `endpoints and its resources were frequently taking ~1.5 min to respond and load the page on both Chrome and non-Chrome browsers for some MacOS users. The Chrome dev tools "Network" tab would show this duration being explicitly "Waiting (TTFB)" during which time there would be no logging in the Django container as well. To start, I enabled Docker's new VirtioFS accelerated directory sharing as well as ran `docker system prune` but didn't see any change in performance. We _did_ however figure out that requests had no delays when using a browser in Incognito mode. Clearing session/3rd party cookies, site data, and any browser extensions in use to try and replicate Incognito mode in the normal browser, however, didn't work. To try and compare differences between these two browser requests I copied both as cURL and ran them from my local terminal a couple times, but in all instances there were no differences or performance issues. Given this, I'm assuming this is a browser-specific issue related to cache or cookies that I just haven't been able to isolate and replicate through cURL. For anyone bumping into this issue, it seems like moving forward in Incognito mode is the easiest way to resolve until we can pin it down.
